src/string.cr

```diff
@@ -3888,7 +3888,7 @@ class String
   # and and non-ascii characters (unicode codepoints > 128), wrapped in quotes.
   #
   # ```
-  # "\u{1f48e} - à la carte\n".dump # => %("\\u1f48e - \\u00e0 la carte\\n")
+  # "\u{1e48e} - à la carte\n".dump # => %("\\u{1E48E} - \\u00E0 la carte\\n")
```

Comment:
changing 1f48e to 1e48e is invalid

Reply:
It's actually the opposite.
1E48E is a meaningless codepoint and 1F48E is a gem
http://www.fileformat.info/info/unicode/char/1F48E/index.htm

Reply:
Oh, I'm silly. :cold_sweat:
Should we use `1F48E` both in here and above example, right?

Reply:
Haven't looked at it but probably